about.css

```diff
@@ -0,0 +1,539 @@
+/* about page */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "lato", sans-serif;
+  min-height: 100vh;
+  background-color: #e5e5e527;
+}
+
+.social-media,
+.navbar {
+  display: none;
+}
+
+.about-box {
+  background: url("./images/home-page.png") no-repeat;
+  background-size: cover;
+  margin-bottom: 50px;
+}
+
+.hamburger {
+  border: none;
+  background: transparent;
+  position: relative;
+  margin-top: 8%;
+  padding-left: 5%;
+}
+
+.hamburger img {
+  width: 60%;
+  height: 70%;
+  margin-bottom: 80%;
+}
+
+.about-intro {
+  position: relative;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-right: 15%;
+  font-weight: normal;
+}
```

Comment:
Please fix the positioning of this content in the desktop version of the `About` page. It is a little slanted to the left when it should be centered on the page.

<img width="1051" alt="Screen Shot 2022-06-22 at 9 10 59" src="https://user-images.githubusercontent.com/82426682/175050657-ed97dac3-bc97-494e-819e-cc6563030a60.png">